code/modules/antagonists/cult/datum_cultist.dm

```diff
@@ -0,0 +1,150 @@
+/datum/antagonist/cultist
+	name = "Cultist"
+	job_rank = ROLE_CULTIST
+	special_role = SPECIAL_ROLE_CULTIST
+	give_objectives = FALSE
+	antag_hud_name = "hudcultist"
+	antag_hud_type = ANTAG_HUD_CULT
+	clown_gain_text = "A dark power has allowed you to overcome your clownish nature, letting you wield weapons without harming yourself."
+	clown_removal_text = "You are free of the dark power suppressing your clownish nature. You are clumsy again! Honk!"
+	clown_text_span_class = "cultitalic"
+	wiki_page_name = "Cultist"
+	var/remove_gear_on_removal = FALSE
+
+/datum/antagonist/cultist/on_gain()
+	create_team() // make sure theres a global cult team
+	..()
+	owner.current.faction |= "cult"
+	add_cult_actions()
+	SEND_SOUND(owner.current, sound('sound/ambience/antag/bloodcult.ogg'))
```

Comment:
Is the starting sound not just part of the antag datum itself?

Reply:
Nope :) None of them do it